iphone/TitaniumKit/TitaniumKit/Sources/API/TiConsole.h

```diff
@@ -14,6 +14,9 @@ JSExportAs(timeLog,
            -(void)timeLog
            : (id)args withData
            : (NSArray *)logData);
+JSExportAs(log,
```

Comment:
I'm not sure this is actually required as the log method id exported by the APIModule parent class...